### ISSUE TITLE
Visual Style Prompting

### ISSUE THREAD

**Issue by edtjulien** [cubiq/ComfyUI_IPAdapter_plus#295]
The code and a demo of Visual Style Prompting is available: https://github.com/naver-ai/Visual-Style-Prompting

My first demo tests gave me the same quality results than in the paper. And not sure I can get the same with IPAdapter. And it's training-free, no additional model needed.

Sounds easy to implement in a node? Like reference_only maybe?

**Comment by laksjdjf** [comment]
I have tried based on the paper, but it did not work as well as the paper. My interpretation seems to be wrong.

https://gist.github.com/laksjdjf/a5fdc2bd4aa8ad79ae16f20f0dc112ef

**Comment by cubiq** [comment]
this is something I'm sure will be implemented directly into the official comfyui soon enough

**Comment by cubiq** [comment]
someone is working on it... doesn't look 100% there though

https://github.com/ExponentialML/ComfyUI_VisualStylePrompting

**Comment by edtjulien** [comment]
Yes! Sounds promising work. Hopefully it's not a Diffusers wrapper.
